data/hai/unfettered 2 war preparations.txt

```diff
@@ -1060,8 +1060,10 @@ mission "<uhai> 15: Modified Beetle"
 			`	"Since this was partly a test-run, what suggestions would you make to improve the ship, so I can make changes to its outfits, before I finally give this ship to you?"`
 			label suggestions
 			choice
-				`	"The ship is fine, but I could use an extra outfit to modify the loadout myself."`
+				`	"No need to modify the loadout, but I could use an extra outfit all the same."`
 				`	"Can you explain to me why you named the ship Hera?"`
+					to display
+						not "unfettered: Hera"
```

Comment:
Do you clear this condition?

Reply:
on purpose no, I think the names of the ships matter enough to keep them